meteor/sofie-server/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -111,15 +111,36 @@ export class MongoEntityConverter {
 	}
 
 	public convertRundowns(mongoRundowns: MongoRundown[]): Rundown[] {
-		return mongoRundowns.map(this.convertRundown)
+		return mongoRundowns.map(this.convertRundown.bind(this))
+	}
+
+	public convertToMongoRundown(rundown: Rundown): MongoRundown {
+		return {
+			externalId: '', // Todo: figure out where the value for this attribute is
+			metaData: { rank: 0 }, // Todo: figure out where the value for this attribute is
+			modified: rundown.getLastTimeModified(),
+			notes: [], // Todo: figure out where the value for this attribute is
+			organizationId: '', // Todo: figure out where the value for this attribute is
+			playlistExternalId: '', // Todo: figure out where the value for this attribute is
+			showStyleBaseId: '', // Todo: figure out where the value for this attribute is
+			showStyleVariantId: '', // Todo: figure out where the value for this attribute is
+			studioId: '', // Todo: figure out where the value for this attribute is
+			timing: { expectedDuration: 0, expectedEnd: 0, expectedStart: 0, type: '' }, // Todo: figure out where the value for this attribute is
+			_id: rundown.id,
+			name: rundown.name,
```

Comment:
Just map the attributes that `convertRundown` is mapping. 
I understand why my previous comment has led you to do this :)

Reply:
All those attributes that you see are required by the `MongoRundown`. This one is the main reason why i previously chose to make e.g the `convertToMongoRundown` return a partial one. 

I have already mapped all the values from `rundown` up with where they originally was extracted, when parsing a `MongoRundown` through the `convertRundown(...)`.